tensorboard/proto/README.md

```diff
@@ -0,0 +1,15 @@
+# TensorFlow Protos
+
+Protobuf files copied from the main TensorFlow repository and used in the case of tensorboard-notf, which builds without a TensorFlow dependency.
+
+## Update process
+
+Copy the proto files from TensorFlow to TensorBoard using the following process:
+
+* git clone tensorflow/tensorboard and tensorflow/tensorboard in ~
+* cd ~/tensorboard
+* ./tensorboard/proto/update.sh
+* git add .
+* git commit -m "Update TensorFlow protos to xxxx"
+
+These were taken from TensorFlow version 1.21.0-dev20181012
```

Comment:
I assume the version should be 1.12, not 1.21?

Reply:
Thanks - fixed.